cime/config/e3sm/config_grids.xml

```diff
@@ -1806,6 +1829,22 @@
       <desc>1-deg with 1/4-deg over CONUS (version 1):</desc>
     </domain>
 
+    <domain name="ne0np4_northamericax4v1">
+      <nx>130088</nx>
+      <ny>1</ny>
+      <file grid="atm|lnd" mask="oRRS15to5">$DIN_LOC_ROOT/share/domains/domain.lnd.northamericax4v1_oRRS15to5.190614.nc</file>
+      <file grid="ice|ocn" mask="oRRS15to5">$DIN_LOC_ROOT/share/domains/domain.ocn.northamericax4v1_oRRS15to5.190614.nc</file>
+      <desc>1-deg with 1/4-deg over North America (version 1):</desc>
+    </domain>
+
+    <domain name="ne0np4_antarcticax4v1">
+      <nx>130088</nx>
+      <ny>1</ny>
+      <file grid="atm|lnd" mask="oRRS15to5">$DIN_LOC_ROOT/share/domains/domain.lnd.antarcticax4v1_oRRS15to5.190614.nc</file>
+      <file grid="ice|ocn" mask="oRRS15to5">$DIN_LOC_ROOT/share/domains/domain.ocn.antarcticax4v1_oRRS15to5.190614.nc</file>
+      <desc>1-deg with 1/4-deg over North America (version 1):</desc>
```

Comment:
Typo here in the description, should be "Antarctica" instead of "North America".